pyspecProcScripts/integral_w_error.py

```diff
@@ -1,14 +1,15 @@
 import pyspecdata as psp
 from .integrate_limits import integrate_limits
 from .simple_functions import select_pathway
+from .calc_error import calc_masked_error
 import logging
 import numpy as np
 
 
 def integral_w_errors(
     s,
     sig_path,
-    error_path,
+    excluded_pathways=None,
```

Comment:
again the default masks out the signal pathway and the f slice in all other pathways (assumes ph cycling noise in f slice in all other pathways)